docs/Makefile

```diff
@@ -1,10 +1,13 @@
 # Makefile for Sphinx documentation
-#
+.DEFAULT_GOAL   = help
+SHELL           = bash
 
 # You can set these variables from the command line.
-SPHINXOPTS      =
-SPHINXBUILD     = sphinx-build
-SPHINXAUTOBUILD = sphinx-autobuild
+SPHINXOPTS      ?=
+
+# Internal variables.
+SPHINXBUILD     = "$(realpath ../.venv/bin/sphinx-build)"
```

Comment:
check:
- [ ] This needs a venv to be set up

Reply:
That is good! If this diverges from what is documented or we do not have that documented at all, let's open a new issue after merge.